packages/ui/v2/core/Stepper.tsx

```diff
@@ -11,8 +12,12 @@ function Stepper<T extends DefaultStep>(props: {
   step: number;
   steps: T[];
   disableSteps?: boolean;
+  t?: TFunction;
 }) {
-  const { t } = useLocale();
+  let { t } = useLocale();
+  if (props.t) {
+    t = props.t;
+  }
```

Comment:
Making the translation configurable from outside in order to be able to use this component from different packages resolving translations differently.